### ISSUE TITLE
Misuse of ShuffleLayer or TrT could fuse this kind of pattern(consecutive shuffle layers)?

### ISSUE THREAD

**Issue by pmixer** [NVIDIA/TensorRT#1197]
## Description

As shuffle layer of a combination of pre-transpose+reshape+post-transpose, sometimes users may use separate shuffle layers for each reshape or transpose, which causing more shuffle layers added to the TrT network.

I'm just wondering whether two or more consecutive shuffle layers may be automatically merged in TrT? Otherwise it might lead to below-expected performance.

## Environment

doesn't matter, as shuffle layer do not change much for different versions.

BTW, could u pls tell if there's a place to check when an op/feature is available in TrT(like TopK is available since TrT 6), as users may  do not prefer to update to newest version in production system frequently.

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

none

## Steps To Reproduce

none

**Comment by pranavm-nvidia** [comment]
Consecutive shuffle layers will be fused by the TensorRT optimizer. 

Regarding your second question, you can look at the `Key Features And Enhancements` section of the [release notes](https://docs.nvidia.com/deeplearning/tensorrt/release-notes/index.html).

**Comment by ttyio** [comment]
Close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
